js/openTrailFeature.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+var openTrailFeature = (function() {
```

Comment:
I just looked at the bundle.js and I understand what you mean now.
I updated to remove them.

Cheers,
Trevor

On Mon, Aug 3, 2015 at 1:31 PM, Sean Garborg <notifications@github.com>
wrote:

> In js/openTrailFeature.js
> <https://github.com/CodeForBoulder/trailsy/pull/11#discussion_r36121719>:
>
> > @@ -0,0 +1,28 @@
> > +"use strict";
> > +
> > +var openTrailFeature = (function() {
>
> I could be missing something, but I think part of the benefit of using a
> build tool that understands modules and transforms modules to closures
> under the hood is you no longer have to use this anonymous closure pattern
> in your code.
>
> Since webpack converts each module file to a closure during the build
> step, can we get rid of these explicit closures?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CodeForBoulder/trailsy/pull/11/files#r36121719>.
>